pyigm/abssys/utils.py

```diff
@@ -13,15 +13,58 @@
 import numpy as np
 from collections import OrderedDict
 
-from astropy import constants as const
 from astropy import units as u
 from astropy.coordinates import SkyCoord
-from astropy.table import QTable
 from astropy.io import ascii
+from astropy.table import Table, Column
 
+from linetools.abund import ions as ltai
 from linetools.spectralline import AbsLine
 from linetools.analysis import absline as ltaa
-from linetools.isgm.abscomponent import AbsComponent
+
+
+def dict_to_ions(idict):
+    """  Manipulate dict into an ion astropy Table
+
+    Parameters
+    ----------
+    idict : dict
+
+    Returns
+    -------
+    table : astropy.Table
+
+    """
+    #  Could probably use add_row or dict instantiation
+    table = None
+    for ion in idict.keys():
+        Zion = ltai.name_ion(ion)
+        if table is None:
+            tkeys = idict[ion].keys()
+            lst = [[idict[ion][tkey]] for tkey in tkeys]
+            table = Table(lst, names=tkeys)
+            # Extra columns
+            if 'Z' not in tkeys:
+                table.add_column(Column([Zion[0]], name='Z'))
```

Comment:
Z is Atomic Number
ZH is metallicity

This should always be the distinction.
Not sure where to document it..